mods/cnc/languages/rules/en.ftl

```diff
@@ -439,126 +491,170 @@ actor-bio-husk-name = Biological Lab (Destroyed)
+
+    It is recommended to mix minigunners with your medium tank armies to deal with enemy infantry.
 
 actor-htnk =
    .name = Mammoth Tank
-   .description = Heavily armored GDI Tank.
-    Can attack Aircraft.
-      Strong vs Everything
+   .description = Heavily armored GDI Tank
+    Can attack Aircraft
+      Strong vs. Everything
+   .encyclopedia = The Mammoth Tank is a crawling battle station, capable of self repair and engaging any threat. It is particularly effective vs enemy armor, but can also destroy small amounts of infantry or aircraft.
+
+    When in an enemy base, put it on aggressive stance to have it carve a path of destruction as it moves.
 
 actor-msam =
    .name = Rocket Launcher
-   .description = Long range rocket artillery.
-      Strong vs all Ground units.
+   .description = Long-range rocket artillery
+      Strong vs. all ground units
+   .encyclopedia = Referred to as “MLRS” by players, this artillery platform fires volleys of rockets at distant targets. It is particularly effective vs light vehicles, and does moderate damage vs infantry, buildings, and heavy armor.
+
+    A strong unit to build vs enemy artillery, as its high bullet velocity and damage to light vehicles makes it the perfect counter to Nod artillery.
 
 actor-mlrs =
-   .name = Mobile S.A.M.
-   .description = Powerful anti-air unit.
-    Cannot attack Ground units.
+   .name = Mobile SAM
+   .description = Powerful anti-air unit
+    Cannot attack ground units
+   .encyclopedia = Generally referred to as MSAM, this is a dedicated anti-air vehicle for Nod when bikes don't cut it. Has a slow missile but with a strong splash.
 
 actor-stnk =
    .name = Stealth Tank
-   .description = Long-range missile tank that can cloak.
-    Can attack Aircraft.
-    Has weak armor. Can be spotted by infantry and defense structures.
-      Strong vs Vehicles, Tanks
-      Weak vs Infantry.
+   .description = Long-range missile tank that can cloak
+    Can attack Aircraft
+    Has weak armor and can be spotted by infantry and defenses
+      Strong vs. Vehicles and Tanks
+      Weak vs. Infantry
+   .encyclopedia = Stealth tanks cloak and uncloak when taking damage or firing their weapons. They can also be detected by defenses in a small radius or infantry in a one cell radius. Their long range AP missiles are excellent for dealing with enemy armor or air.
+
+    Can crush infantry in a pinch.
 
 actor-mhq =
    .name = Mobile HQ
    .description = Mobile base of operations
 
 actor-truck =
-   .description = Transports cash to other players.
-      Builds fast
+   .description = Transports cash to other players
+      Builds quickly
       Unarmed
    .name = Supply Truck
+   .encyclopedia = Supply trucks are a convenient way to share cash when your ally is out of money, or you can't spend yours fast enough. Costs and delivers 1k money but builds 2x as fast.
```

Comment:
Not sure we should hardcode the cash value, would be easy for this to go out of sync if the value is ever changed.